### PR TITLE
Add "DWG/DXF Import..." description

### LINKED ISSUE #1579: [FEATURE] DWG import into geopackage using libdxfrw
Original commit: https://github.com/qgis/QGIS/commit/d5c6b977eb9d58484525953c753b164957b51a16 by jef-n

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by DelazJ**
Fixes #1579

### CODE REVIEW

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 1]
```diff
@@ -363,9 +363,34 @@ to the project. Layers are added with random style properties.
...
+  ``curved`` one.
+
+After the :file:`.dwg` or :file:`.dxf` data is imported into the GeoPackage
+database the list in the lower half of the dialog is populated with the layer
+list from the imported file. There you can select which layers to import into
+the QGIS project. A top level group is set for the drawing file and inside,
+another group for each of the selected layers in the list. Each subgroup
+contains vector layers for the point, line, label and area features of the
+respective layer. The style of each layer is setup so that it resembles the look
+it originally had in *CAD.
```
Comment (SrNetoChan) [inline source/docs/user_manual/managing_data_source/opening_data.rst]: missing * ?

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 2]
```diff
@@ -363,9 +363,34 @@ to the project. Layers are added with random style properties.
...
+  drawing file content.
+* The |checkbox| :guilabel:`Expand block references` helps you copy the
+  transformed contents of the corresponding block for each block reference.
+* the |checkbox| :guilabel:`Use curves` promotes the layers geometry type to a
+  ``curved`` one.
+
+After the :file:`.dwg` or :file:`.dxf` data is imported into the GeoPackage
+database the list in the lower half of the dialog is populated with the layer
+list from the imported file. There you can select which layers to import into
+the QGIS project. A top level group is set for the drawing file and inside,
```
Comment (SrNetoChan) [inline source/docs/user_manual/managing_data_source/opening_data.rst]: I would make a reference to the Group name label.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 3]
```diff
@@ -363,9 +363,34 @@ to the project. Layers are added with random style properties.
 .. need to be tested with dwg. How does dwg format behave when added to QGIS?
```
[the remark this answers is not in the document]
Comment (jef-n) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: GDAL now has a opencad driver that reads DWG - but reading through OGR will not address symbology alike.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 4]
```diff
@@ -363,9 +363,34 @@ to the project. Layers are added with random style properties.
...
 
 To keep the dxf/dwg structure and its symbology in QGIS, you may want to
-use the dedicated :menuselection:`DWG/DXF Import...` tool.
+use the dedicated :menuselection:`Project --> DWG/DXF Import...` tool. Indeed,
+the :guilabel:`DWG/DXF Import` dialog offers you to import in a GeoPackage
```
[the remark this answers is not in the document]
Comment (jef-n) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: imports into GeoPackage

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 5]
```diff
@@ -363,9 +363,34 @@ to the project. Layers are added with random style properties.
...
+
+In the dialog, you have to:
+
+* Input a location for a GeoPackage file, that will be created to store the
+  DWG/DXF content to.
+* Specify which coordinate system the data in the DWG data is in.
+* Then use the **[Import]** button to select the DWG/DXF file to use (one per
+  geopackage). The GeoPackage database will be automatically populated with the
+  drawing file content.
+* The |checkbox| :guilabel:`Expand block references` helps you copy the
```
[the remark this answers is not in the document]
Comment (jef-n) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: @SrNetoChan correct.

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: @jef-n, is it in the minimal version required for 3.0. I mean, could we say that dwg can be opened in 3.0 or should this statement be conditioned by a GDAL version?

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Could someone come with a text, please? I'm far from CAD for maaannny years now and just pick the description from the "commit" help message https://github.com/qgis/QGIS-Documentation/issues/1580#issuecomment-267781898 (btw, really great to ease writers work. Thanks)

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 1 above)
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Picked from aforementioned description (i guess it was instead targeting softwares like LibreCAD, AutoCAD... ). Simply forgot that * is a special character and Travis would not like it. 
Will change it.

**Comment by DelazJ** [comment]
Thanks for your reviews

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (jef-n) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: No, DWG import was introduced in 2.18.  And our DWG import is independant of GDAL's DWG support (which didn't exist when the QGIS' DWG import was implemented; QGIS DWG support aims to also reproduce the symbology and layer structure from *CAD; importing via GDAL would only produce simple points, lines, polygons)

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 1 above)
Comment (SrNetoChan) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: You need to escape the *. I think \\\* will do

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 6]
```diff
@@ -352,20 +352,50 @@ Additionally, you can enable:
...
+database any element of the drawing file.
+
+In the dialog, you have to:
+
+* Input a location for a GeoPackage file, that will be created to store the
+  DWG/DXF content to;
+* Specify which coordinate system the data in the DWG data is in;
+* Then use the **[Import]** button to select the DWG/DXF file to use (one per
+  geopackage). The GeoPackage database will be automatically populated with the
+  drawing file content. Depending on the size of the \*CAD file, this could
```
Comment (DelazJ) [inline source/docs/user_manual/managing_data_source/opening_data.rst]: I think the automatic import is a UX killer: select the wrong CAD file and you have to wait for a long time: i think people expect to first add the cad layer (btw we should use qgsfilewidget for both data loader) and then consciously choose to import into the db.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 7]
```diff
@@ -352,20 +352,50 @@ Additionally, you can enable:
...
+  blocks into normal elements;
+* the |checkbox| :guilabel:`Use curves` promotes the output layers geometry type
+  to a ``curved`` one.
+
+After the :file:`.dwg` or :file:`.dxf` data is imported into the GeoPackage
+database the frame in the lower half of the dialog is populated with the list of
+layers from the imported file. There you can select which layers to add to the
+QGIS project:
+
+* At the top, set a :guilabel:`Group name` to group the drawing files in the
```
Comment (DelazJ) [inline source/docs/user_manual/managing_data_source/opening_data.rst]: why is it required to add a group name (ok is disabled until set)?

**Comment by DelazJ** [comment]
Status?
I'll open bug report for what look like issue to me and let undocumented the behavior I do not understand

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 7 above)
Comment (jef-n) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: the layers from the DWG appear in a group.

**Pull request merged**